sysutils/watchman/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           cmake 1.1
 PortGroup           boost 1.0
 PortGroup           cargo_fetch 1.0
 
-github.setup        facebook watchman 2021.09.20.00 v
+github.setup        facebook watchman 2022.01.10.00 v
```

Comment:
this appears to close https://trac.macports.org/ticket/64384 and if so, that should be added to the body of the commit message so that it happens automatically when merging

Reply:
Oops, did not see that